packages/react-ui-core/src/Form/RadioGroup.js

```diff
@@ -75,14 +75,16 @@ export default class RadioGroup extends Component {
 
   renderRadioButton(index, fieldProps) {
     const props = pick(this.props, ['name', 'hideInputElement', 'orientation'])
+    const label = { 'data-tid': 'radiogroup-radiobutton', text: fieldProps.label }
     return (
       <RadioButton
-        data-tid="radiogroup-radiobutton"
         key={`${this.id}-${index}`}
         checked={this.state.value === fieldProps.value}
         onChange={this.handleValueChange}
+        label={label}
```

Comment:
You cannot change things like this in `react-ui-core`. This affects _everything_ that's using this component and you're now changing the structure of this label completely.  How does one add anything to label now? Won't this get overrided if I pass in a `label` prop?

Reply:
Also, this is now *forcing* a label which isn't always needed.